tests/shardddl1/run.sh

```diff
@@ -499,6 +499,7 @@ function DM_DropAddColumn_CASE() {
 function DM_DropAddColumn() {
 	for i in $(seq 0 5); do
 		echo "run DM_DropAddColumn case #${i}"
+		sleep 2
```

Comment:
why sleep here

Reply:
just try.. i don't know why this case is unsteable, it  passed every time on my own machine...

Reply:
we'll take a look tomorrow